### PR TITLE
[docs]: Update EAS Update getting started

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Refs https://github.com/expo/expo/pull/18947#issuecomment-1247983592

# How

This PR  groups all the commands, and add a description above instead of each step separately. It also updates some verbiage in the same section.

# Test Plan

Changes can be tested by running `docs/` locally and visiting: http://localhost:3002/eas-update/getting-started/#create-a-project

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/eas-update/getting-started.md [hunk 1]
```diff
@@ -34,25 +34,26 @@ Install EAS and Expo CLIs by running:
 
 Create a project by running:
 
-<Terminal cmd={['$ npx create-expo-app']} />
+<Terminal cmd={['$ npx create-expo-app && ']} />
```
Comment (Simek) [inline docs/pages/eas-update/getting-started.md:37]:
```suggestion
<Terminal cmd={['$ npx create-expo-app']} />
```

Looks like unintended change.

File: docs/pages/eas-update/getting-started.md [hunk 2]
```diff
@@ -34,25 +34,26 @@ Install EAS and Expo CLIs by running:
...
+'',
+'# Initialize your project with EAS Update',
+'$ eas update:configure',
+'',
+'# Set up the configuration file for builds',
+'$ eas build:configure',
+]} cmdCopy="expo install expo-updates && eas update:configure && eas build:configure" />
 
-This command will create a file named **eas.json**.
+After running these commands, **eas.json** is created in the root directory of your project.
```
Comment (Simek) [inline docs/pages/eas-update/getting-started.md:54]: Something seems off to me in this rewritten sentence. What do you think about:

```suggestion
After running these commands, **eas.json** file should be present in the root directory of your project.
```

or

```suggestion
After running these commands, **eas.json** file will be created in the root directory of your project.
```

**Review by Simek — APPROVED** [review]
Left two small suggestions, beside that LGTM! 👍

File: docs/pages/eas-update/getting-started.md (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/eas-update/getting-started.md:37]: Yes it is. Thanks for catching this!

**Pull request merged**